Formula/portcullis.rb

```diff
@@ -0,0 +1,80 @@
+class Portcullis < Formula
+  include Language::Python::Virtualenv
+  # cite "https://www.biorxiv.org/content/early/2017/11/10/217620"
+  desc "Genuine splice junction prediction from BAM files"
+  homepage "https://github.com/maplesond/portcullis"
+  url "https://github.com/maplesond/portcullis/archive/Release-1.1.0.tar.gz"
+  sha256 "872c0dbd7515229ecc22c9bdcd72eb78dfe93a3c0bfd14af52c448c142fe892a"
+  head "https://github.com/maplesond/portcullis.git"
+
+  bottle do
+    root_url "https://linuxbrew.bintray.com/bottles-bio"
+    sha256 "9eca4a8956da34111ac749f86426c8147916add152d89b13acefad48c2e87691" => :sierra_or_later
+    sha256 "fdd8b8f9452eac884f6772afc651c141c89885902c284e002e3e2b25a2ce5f4e" => :x86_64_linux
+  end
```

Comment:
A new formula shouldn't yet have bottles. You can remove this bottle block.

Reply:
ok, I'll remove this.